### PR TITLE
Add custom mutations support

### PR DESCRIPTION
**Pull request by pucinsk**
### Defining custom mutations

You can define your custom mutations by adding instance method, like this:

```ruby
class User
  include ActiveGraphql::Model
  active_graphql do |c|
    c.attributes :id, :first_name, :last_name
  end
  def update_name(first_name, last_name)
    mutation_method(:update_name, input: { first_name: 'Fancy', last_name: 'Pants' })
  end
end
User.last.update_name('Fancy', 'Pants')
```

This will produce GraphQL:
```graphql
mutation {
  updateName(id: 99, input: { firstName: 'Fancy', lastName: 'Pants' }) {
    id
    firstName
    lastName
    ...
  }
}
```

### CODE REVIEW

File: lib/active_graphql/model.rb [hunk 1]
```diff
@@ -37,16 +37,19 @@ def initialize(attributes)
         @attributes = attributes.deep_transform_keys { |it| it.to_s.underscore.to_sym }
       end
 
-      def update(params)
-        action_name = "update_#{self.class.active_graphql.resource_name}"
+      def mutation_method(action_name, params = {})
```
Comment (povilasjurcys) [inline lib/active_graphql/model.rb]: it does not returns `method` so I would recommend to name it to something like `mutation`

```suggestion
      def mutation(action_name, params = {})
```

File: docs/model.md [hunk 2]
```diff
@@ -331,6 +331,38 @@ query {
 }
 ```
 
+### Defining custom mutations
```
Comment (povilasjurcys) [inline docs/model.md]: can you keep documentation consistent by adding method name as a header?

File: lib/active_graphql/model.rb (hunk 1 above)
Comment (pucinsk) [reply lib/active_graphql/model.rb]: Perfect name for new born! :rocket:

File: docs/model.md (hunk 2 above)
Comment (pucinsk) [reply docs/model.md]: constitency is saved!

**Pull request merged**